src/release/breaking-changes/flutter-gradle-plugin-apply.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Deprecated imperative apply of Flutter Gradle Plugin
+description: >
+  How to migrate your Android app's Gradle build files to the new, declarative
+  format.
+---
+
+## Summary
+
+To build a Flutter app for Android, the Flutter Gradle plugin must be applied.
+Historically, this was done imperatively with Gradle's
+[legacy, imperative apply method][].
+
+In Flutter 3.16, support has been added for applying these plugins with Gradle's
+[declarative plugins {} block][], and it is the recommended approach. Since
+Flutter 3.16, projects generated with `flutter create` use the new apply method
+of applying Gradle plugins. Older projects, however, should be migrated
+manually.
+
+To learn about advantages the new declarative apply syntax has over the old
+imperative apply syntax, see [Gradle docs about plugins {} block][plugins block].
```

Comment:
Should this be an actual link to the Gradle docs?

Reply:
It is. At the bottom of the page there's:

```
[plugins block]: https://docs.gradle.org/current/userguide/plugins.html#plugins_dsl_limitations
```